docs-assets/app/app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -32,12 +33,12 @@ public function panel(Panel $panel): Panel
             ->colors([
                 'primary' => Color::Amber,
             ])
-            ->discoverResources(in: app_path('Filament/Resources'), for: 'App\\Filament\\Resources')
-            ->discoverPages(in: app_path('Filament/Pages'), for: 'App\\Filament\\Pages')
+            ->discoverResources(in: app_path('Filament/Resources'), for: Filament::namespaceFor('Filament\\Resources'))
+            ->discoverPages(in: app_path('Filament/Pages'), for: Filament::namespaceFor('Filament\\Pages'))
```

Comment:
Also, no generated files should contain calls to `Filament::namespaceFor()` or `app()->getNamespace()` - we should insert the namespace at the time of file generation. Otherwise it looks messy in the user's codebase, especially for 99.9% of users who do not use this feature.